velox/type/tests/ConversionsTest.cpp

```diff
@@ -921,20 +921,41 @@ TEST_F(ConversionsTest, toRealAndDouble) {
     // When TRUNCATE = false.
     testConversion<std::string, float>(
         {
-            "1.7E308",
-            "1.",
-            "1",
-            "infinity",
-            "-infinity",
-            "InfiNiTy",
-            "-InfiNiTy",
-            "nan",
-            "nAn",
+            "1.7E308",   "1.",           "1",
+            ".1324",     "1.2345678E19", "1.2345678E8",
+            "1.0E7",     "12345.0",      "0.001",
+            "1.2E-4",    "0.0",          "-0.0",
+            "-1.2E-4",   "-0.001",       "-12345.0",
+            "-1.0E7",    "-1.2345678E8", "-1.2345678E19",
+            " 123",      "123 ",         " 123 ",
+            "infinity",  "-infinity",    "InfiNiTy",
+            "-InfiNiTy", "Inf",          "-Inf",
+            "nan",       "nAn",
         },
         {
             kInf,
             1.0,
             1.0,
+            0.1324,
+            12345678000000000000.0,
```

Comment:
Would you confirm that this test is passing without the changes, i.e. this change doesn't change semantics?

Perhaps, extract this change into a separate PR. We can land that first.

Reply:
I ran these tests before the modification and they all passed. If necessary, I will submit it in a new PR.

Reply:
Got it. A separate PR to extend existing test would be nice. Thanks. This way there will be no doubt that the changes are not introducing any behavior difference.

Reply:
I have submitted a new [PR](https://github.com/facebookincubator/velox/pull/9148).